src/jsonify/parse.ts

```diff
@@ -149,8 +153,8 @@ function unpack(
     hydrated[idx] = actual;
 
     const keys = Object.keys(current);
-    for (let i = 0; i < keys.length; i++) {
-      const key = keys[i];
```

Comment:
This change isn't necessary to fix the issue. As a side not in most browsers using a standard for loop is still much faster on arrays than a `for-of` loop.

Reply:
@marvinhagemeister Perfectly, I will remove this part from the implementation :smile: